samza-core/src/main/scala/org/apache/samza/util/FileUtil.scala

```diff
@@ -26,9 +26,13 @@ import java.nio.file._
 import java.util.zip.CRC32
 
 class FileUtil extends Logging {
+  val VERSION = 1
```

Comment:
Where is this used? If not used in the actual serialized data format (might be difficult to do now without breaking backwards compatibility), let's remove from class.

Reply:
Removed thanks, was initially intending it to be read after the first segment but I think that introduces potential issues regarding reading of the first block read, for full backwards compatibility